src/features/user/Settings/ImpersonateUser/ImpersonateUser.module.scss

```diff
@@ -32,3 +32,31 @@
   max-width: 208px;
   text-align: center;
 }
+// Loader for Buttons
+.spinner {
+  border: 2px solid #f3f3f3;
+  border-radius: 50%;
+  border-top: 2px solid transparent;
+  width: 16px !important;
+  height: 16px !important;
+  -webkit-animation: spin 1s linear infinite; /* Safari */
+  animation: spin 1s linear infinite;
+}
+
+@-webkit-keyframes spin {
```

Comment:
since `-webkit-` for WebKit-based browsers (Safari and older versions of Chrome). will it work in Mozilla and other browsers ?

Reply:
Yes. Also, this is the same code as written everywhere else for spinner class. I saw that the code for spinner has been repeated at multiple places and probably should be kept common in the future.